src/main/java/step3/study/domain/Car.java

```diff
@@ -1,24 +1,25 @@
 package step3.study.domain;
 
+import step3.study.util.NumberGenerator;
+
 public class Car {
-    private String distance = "";
+    private Position position;
     private static final int MOVE_CONDITION = 4;
 
     public Car() {
     }
 
-    public Car(String distance) {
-        this.distance = distance;
+    public Car(Position position) {
+        this.position = position;
     }
 
-    public String move(int randomNumber) {
-        if (MOVE_CONDITION <= randomNumber) {
-            this.distance += "-";
+    public void move(NumberGenerator numberGenerator) {
+        if (MOVE_CONDITION <= numberGenerator.createNumber()) {
+            position.increase();
         }
-        return this.distance;
     }
 
-    public String getDistance() {
-        return this.distance;
+    public int position() {

```

Comment:
```suggestion
    public int getPositionValue() {
```

Position 객체와, 실제 position 값이랑 코드를 읽는 사람이 어떻게 헷갈리지 않을 수 있는지에 대한 배려가 있으면 좋을 것 같습니다!